k8s/agent-scaler/agent-scaler.sh

```diff
@@ -6,23 +6,35 @@ check_delay=${CHECK_DELAY:-10}
 # Kubernetes 1.21 supports jobs TTL controller, which cleans up jobs automatically
 # see https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/
 use_k8s_ttl_controller=${USE_K8S_TTL_CONTROLLER:-false}
+restart_agents_on_boot=${RESTART_AGENTS_ON_BOOT:-false}
 
 namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
 
+if [ -z "${AGENT_TAG}" ]; then
+    # if AGENT_TAG is not set through --build-arg,
+    # use the default agent image and tag
+    export AGENT_TAG="foundationdb/joshua-agent:latest"
+fi
```

Comment:
```suggestion
# if AGENT_TAG is not set through --build-arg,
# use the default agent image and tag
export AGENT_TAG=${AGENT_TAG:-"foundationdb/joshua-agent:latest"}
```

This would be the same correct?